tests/unit/validate-test.js

```diff
@@ -81,6 +82,17 @@ test('runs all validations', function(assert) {
   return promise;
 });
 
+test('it can be mixed into an Ember Object', function(assert) {
+  let Being = EmberObject.extend(EmberValidations, {
+    container: getOwner(this), /* test fails if removed */
```

Comment:
would it not be possible to use `Ember.setOwner` on your object so it can use the `owner` API instead of injecting a psuedo container?